### PR TITLE
Add discord

### PR DESCRIPTION
**Pull request by goetas**
Preview at https://www---pr-285-m6ay72q-ovkfaqrdltr5s.us.platform.sh/

### CODE REVIEW

File: source/_layouts/base.twig [hunk 1]
```diff
@@ -93,6 +93,9 @@
                 <li class="site_footer_links__item site_footer_links__item--irc">
                     <a class="site_footer_links__link" href="/irc/">Chat on IRC Channel</a>
                 </li>
+                <li class="site_footer_links__item site_footer_links__item--discord">
+                    <a class="site_footer_links__link" href="https://disboard.org/server/788815898948665366">Talk on Discord</a>
```
Comment (Crell) [inline source/_layouts/base.twig:97]:
```suggestion
                    <a class="site_footer_links__link" href="https://disboard.org/server/788815898948665366">Discuss on Discord</a>
```

**Comment by Crell** [comment]
I think we can get rid of the IRC link.  No one uses it, at all.  And then we can reuse that icon for Discord.

**Review by Jean85 — CHANGES_REQUESTED** [review]
CSS is not applying color to the Discord icon

immagine

**Comment by goetas** [comment]
- Added discord icons
- updated text

**Review by Jean85 — APPROVED** [review]

**Pull request merged**